src/devices/zemismart.ts

```diff
@@ -105,6 +105,40 @@ const definitions: Definition[] = [
             await reporting.onOff(device.getEndpoint(2));
         },
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_7eue9vhc', '_TZE200_bv1jcqqu']),
+        model: 'ZM25RX-08/30',
+        vendor: 'Zemismart',
+        description: 'Tubular motor',
+        onEvent: tuya.onEvent(),
+        configure: tuya.configureMagicPacket,
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        options: [exposes.options.invert_cover()],
+        exposes: [
+            e.text('work_state', ea.STATE),
+            e.cover_position().setAccess('position', ea.STATE_SET),
+            e.battery(),
+            e.enum('program', ea.SET, ['SET BOTTOM', 'SET UPPER', 'RESET']).withDescription('Set the upper/bottom limit'),
+            e.enum('program', ea.SET, ['UPPER', 'UPPER MICRO', 'LOWER', 'LOWER MICRO']).withDescription('Steps control (ignores set limit)'),
+            e.enum('motor_direction', ea.STATE_SET, ['NORMAL', 'REVERSED']).withDescription('Motor direction'),
+        ],
+        meta: {
+            tuyaDatapoints: [
+                [1, 'state', tuya.valueConverterBasic.lookup({'OPEN': tuya.enum(2), 'STOP': tuya.enum(1), 'CLOSE': tuya.enum(0)})],
+                [2, 'position', tuya.valueConverter.coverPosition],
+                [3, 'position', tuya.valueConverter.coverPosition],
+                [5, 'motor_direction', tuya.valueConverterBasic.lookup({'NORMAL': tuya.enum(0), 'REVERSED': tuya.enum(1)})],
```

Comment:
Can you move this value converter inside `lib/tuya.ts`? Then it can also be used by `TS0601_cover_4`, same goes for the expose